### PR TITLE
Revert #25047 - [SQLite] Bump and use byte[] variants added to blob_open for performance

### PR DESCRIPTION
**Pull request by OmarTawfik**
Reverts #25047 as it is causing NGen issues in official builds.
We need to consolidate different versions of that binary that ship into VS.

cc @tannergooding can you please review?

### CODE REVIEW

**Comment by tannergooding** [comment]
FYI. @jinujoseph and @jasonmalinowski as well.

**Comment by tannergooding** [comment]
@OmarTawfik (or @jinujoseph) could we get a bug logged tracking getting this re-enabled.

If 1.1.5 works, we can just move to that. Otherwise, we need to ensure the appropriate binding redirects are added to VS.

In either case, we should also ensure we are consuming the shared copy, rather than inserting our own.

File: build/Targets/Packages.props [hunk 1]
```diff
@@ -230,7 +230,7 @@
     <SystemXmlXmlDocumentVersion>4.3.0</SystemXmlXmlDocumentVersion>
     <SystemXmlXmlSerializerVersion>4.3.0</SystemXmlXmlSerializerVersion>
     <SystemXmlXPathXDocumentVersion>4.3.0</SystemXmlXPathXDocumentVersion>
-    <SQLitePCLRawbundle_greenVersion>1.1.11</SQLitePCLRawbundle_greenVersion>
+    <SQLitePCLRawbundle_greenVersion>1.1.2</SQLitePCLRawbundle_greenVersion>
```
Comment (heejaechang) [inline build/Targets/Packages.props]: we only need to revert this. not whole thing...?

Comment (tannergooding) [reply build/Targets/Packages.props]: The other changes require this change

Comment (heejaechang) [reply build/Targets/Packages.props]: are we reverting whole thing to be safe?

Comment (tannergooding) [reply build/Targets/Packages.props]: (As I understand it)

Comment (Therzok) [reply build/Targets/Packages.props]: The `ReadBlob(byte[])` APIs were added in 1.1.10, so it would not build otherwise.

**Comment by OmarTawfik** [comment]
@dotnet-bot test windows_release_unit32_prtest

**Pull request merged**